contrib/debian/bitcoin-qt.desktop

```diff
@@ -1,8 +1,9 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Bitcoin
-Comment=Bitcoin P2P Cryptocurrency
+Comment=Connect to the Bitcoin P2P Network
 Comment[fr]=Bitcoin, monnaie virtuelle cryptographique pair à pair
+Comment[de]=Bitcoin, eine virtuelle kryptographische peer-to-peer Währung
```

Comment:
I guess this should match the English one. I.e.


"Verbinde mit dem Bitcoin peer-to-peer Netzwerk" (oder ähnlich)